usr/src/uts/i86pc/io/vmm/vmm.c

```diff
@@ -1234,20 +1238,20 @@ vm_set_register(struct vm *vm, int vcpuid, int reg, uint64_t val)
 	return (0);
 }
 
-static boolean_t
+static bool
```

Comment:
Yes -- we've just been a bit reluctant about adopting them -- mostly since shared headers between kernel and user land could have pre C99 user land consumers. However for purely in-kernel interfaces (and especially internal module interfaces) bool is fine -- I'd even go so far as to decree 'consumers of the vmm device in user land must be C99 or newer'.